### DISCUSSION TITLE
Support for multiple tabs in the playlist window

### DISCUSSION THREAD

**Discussion by uctopuk** [smplayer-dev/smplayer#614] [Ideas]
Hello. It seems to me that it would be a great idea to implement the ability to create tabs in the playlist window, as is implemented, for example, in PotPlayer on Windows. I have a need to use several playlists in parallel, and it is very inconvenient that this useful feature is still not in this wonderful player. To date, SMPlayer almost completely satisfies my needs and pleases with the flexibility of its settings. The only thing I lack for complete happiness is the ability to keep several playlists open at the same time and freely switch between them. I hope my idea will be supported by a respected community and developers! Thanks to all! Peace and goodness to all!

![PotPlayer-06](https://user-images.githubusercontent.com/120328955/206918042-d6d9ba60-7495-4d0c-b48b-d141efb20b9d.png)

**Comment by madgabz** [comment]
THIS! Multiple Tabbed playlists! And, auto-filling a playlist with [newer] files from specific locations/dirs.
Usecase: I travel a lot, and to view my favourite youtube playlists, i use yt-dlp to auto download to my NAS, from where i can view or transfer to my laptop. Assigning each playlist to its own playlist/tab would make it soo much easier to quickly switch to new content. Potplayer in windows was almost perfect for this... but.. windows!
